helpers/utils/b64.js

```diff
@@ -0,0 +1,31 @@
+export const decodeB64StringToArrayBuffer = (input) => {
+  const bytes = Math.floor((input / 4) * 3)
```

Comment:
This should be `input.length`, not `input`, otherwise you just get NaN and `bytes` is always interpreted as 0.
```suggestion
  const bytes = Math.floor((input.length / 4) * 3)
```

Reply:
Oooh, good catch!